apollo-sample/src/main/java/com/example/apollographql/sample/PostsService.java

```diff
@@ -0,0 +1,14 @@
+package com.example.apollographql.sample;
+
+import com.apollographql.android.api.graphql.Response;
+import com.example.AllPosts;
+import com.example.Upvote;
+
+import io.reactivex.Observable;
+import retrofit2.http.Body;
+import retrofit2.http.POST;
+
+interface PostsService {
```

Comment:
nit: rename to `FrontPageService` since that's the schema we're using: https://github.com/apollographql/frontpage-server